locales/locales.ini

```diff
@@ -20,3 +20,6 @@
 
 [nl]
 @import url(notes.nl.properties)
+
+[uk]
+@import url (notes.uk.properties)
```

Comment:
btw, with Firefox WebIDE you can install and test this app on the FxOS simulator to ensure that strings are loading and displaying properly